playwright.config.ts

```diff
@@ -0,0 +1,21 @@
+import { PlaywrightTestConfig } from '@playwright/test';
+
+// Config file docs: https://playwright.dev/docs/test-configuration
+const config: PlaywrightTestConfig = {
+  webServer: {
+    command: './node_modules/.bin/next start',
+    port: 3000,
+  },
```

Comment:
let's also add `stdout: 'pipe'` (default is `'ignore'`, which discards stdout from the process)

this can be useful for debugging

Reply:
done in:

- https://github.com/upleveled/next-js-portfolio-web/commit/d8f20f37c71ea021dbd0a3a8f26541df1b2265ac
- https://github.com/upleveled/next-starter-peacock/commit/724d53cad776d9016b93856ec5acb1a48bf08c22
- [upleveled/portfolio-cms@`c32fe2a` (#112)](https://github.com/upleveled/portfolio-cms/pull/112/commits/c32fe2ae9c20be0740746795df9aa4dfbeae37cb)
- [upleveled/next-portfolio-dev@`c538b0b` (#87)](https://github.com/upleveled/next-portfolio-dev/pull/87/commits/c538b0b411de0dcba4c819f90622f264db081e5d)